code/modules/procedural_mapping/mapGenerators/asteroid.dm

```diff
@@ -19,9 +19,11 @@
 //Monsters
 /datum/map_generator_module/splatter_layer/asteroid_monsters
 	spawnableTurfs = list()
-	spawnableAtoms = list(/mob/living/simple_animal/hostile/asteroid/basilisk = 10, \
+	spawnableAtoms = list(
+		/mob/living/basic/mining/goliath/ancient = 10,\
+		/mob/living/simple_animal/hostile/asteroid/basilisk = 10, \
 		/mob/living/simple_animal/hostile/asteroid/hivelord = 10, \
-		/mob/living/simple_animal/hostile/asteroid/goliath = 10)
+		)
```

Comment:
we don't need the backslashes here, we may have needed them in older versions of DM but we just don't need them anymore. also your end parenthesis was off

```dm
	spawnableAtoms = list(
		/mob/living/basic/mining/goliath/ancient = 10,
		/mob/living/simple_animal/hostile/asteroid/basilisk = 10,
		/mob/living/simple_animal/hostile/asteroid/hivelord = 10,
	)
```